test/systemtests/docker_test.go

```diff
@@ -603,6 +603,38 @@ func (d *docker) checkSchedulerNetworkCreated(nwName string, expectedOp bool) er
 	return err
 }
 
+func (d *docker) checkSchedulerNetworkOnNodeCreated(nwNames []string, n *node) error {
+	ch := make(chan error, 1)
```

Comment:
Actually that may not happen. I would have to test to be sure. Let's leave it alone instead of sending that through CI again.